x = 10

def my_function():
    x = x + 5
    print(x)

my_function() # It will raise exception since it x has no value when it tries to reassign its value.